explorer/templates/explorer/query_favorites.html

```diff
@@ -0,0 +1,25 @@
+{% extends "explorer/base.html" %}
+{% load i18n %}
+
+{% block sql_explorer_content %}
+    <h2>{% trans "Favorite Queries" %}</h2>
+    <div class="table-responsive">
+        <table class="table table-striped query-list">
+            <thead>
+            <tr>
+                <th>{% trans "Query" %}</th>
+            </tr>
+            </thead>
+
+            <tbody>
+            {% for favorite in favorites %}
+                <tr>
+                    <th>
+                        <a href="{% url 'query_detail' favorite.query.id %}">{{ favorite.query.title }}</a>
+                    </th>
+                </tr>
+            {% endfor %}
```

Comment:
A bit of explanation would work on here if no favorites exist yet.

```suggestion

            {% empty %}
                <tr>
                    <th>
                        {% trans "No favorite queries added yet." %}
                    </th>
                </tr>
            {% endfor %}
```